app/renderer/src/components/App.js

```diff
@@ -24,7 +25,7 @@ const base = style({
   overflow: 'hidden',
 });
 
-const App = ({ theme }) => {
+const App = ({ q, theme, onQueryChange, onQueryReset }) => {
```

Comment:
I would go as far as to remove the destruct
one thing I like with not using destruct, is that the forces SOC (cough propTypes validation)

Reply:
Great point